packages/core/src/fields/types/relationship/views/RelationshipSelect.tsx

```diff
@@ -1,43 +1,9 @@
-/** @jsxRuntime classic */
-/** @jsx jsx */
-
 import 'intersection-observer'
```

Comment:
can we add 'use client' here, this is harmless in non ssr but useful in SSR with upcoming changes.

Reply:
@gautamsi any reason not to do this in your PR rather than this one?

Reply:
I can do that

Reply:
Totally on board with SSR friendly! This PR is pretty massive in scope, let's focus on app-router etc. after it's landed.

Reply:
Looking forward to get this design uplift, would love to see this coming. wait is killing!!